packages/xstate-react/src/createActorContext.ts

```diff
@@ -32,49 +32,52 @@ type ToMachinesWithProvidedImplementations<TMachine extends AnyStateMachine> =
       >
     : never;
 
-export function createActorContext<TMachine extends AnyActorLogic>(
-  actorLogic: TMachine,
-  interpreterOptions?: InterpreterOptions<TMachine>,
+export function createActorContext<TLogic extends AnyActorLogic>(
+  actorLogic: TLogic,
+  interpreterOptions?: InterpreterOptions<TLogic>,
   observerOrListener?:
-    | Observer<SnapshotFrom<TMachine>>
-    | ((value: SnapshotFrom<TMachine>) => void)
+    | Observer<SnapshotFrom<TLogic>>
+    | ((value: SnapshotFrom<TLogic>) => void)
```

Comment:
We have no tests utilizing those 2 arguments. I wonder if we need `observerOrListener` here - it feels weird here really, since this such a listener feels instance-specific and `createActorContext` doesn't actually create an instance of an actor.